redis/commands/core.py

```diff
@@ -3350,9 +3350,13 @@ def sinterstore(
         args = list_or_args(keys, args)
         return self.execute_command("SINTERSTORE", dest, *args)
 
-    def sismember(self, name: str, value: str) -> Union[Awaitable[bool], bool]:
+    def sismember(
+        self, name: str, value: str
+    ) -> Union[Awaitable[Union[Literal[0], Literal[1]]], Union[Literal[0], Literal[1]]]:
         """
-        Return a boolean indicating if ``value`` is a member of set ``name``
+        Return whether ``value`` is a member of set ``name``:
```

Comment:
@shacharPash @chayim What's the point of introducing this breaking change and forcing users to struggle with "1"/"0" comparisons?

JFYI:
```
>>> "Bang!" if "0" else "OK!"
'Bang!'
```

Reply:
@uglide To be consistent with smismember (see #2622)